beacon-chain/rpc/eth/v1/validator/validator.go

```diff
@@ -238,7 +240,71 @@ func (vs *Server) SubmitAggregateAndProofs(ctx context.Context, req *v1.SubmitAg
 // SubmitBeaconCommitteeSubscription searches using discv5 for peers related to the provided subnet information
 // and replaces current peers with those ones if necessary.
 func (vs *Server) SubmitBeaconCommitteeSubscription(ctx context.Context, req *v1.SubmitBeaconCommitteeSubscriptionsRequest) (*emptypb.Empty, error) {
```

Comment:
I could not simply embed the v1alpha1 server and execute its function equivalent because I need to assign aggregator to subnet in the middle of the function (see line 303).